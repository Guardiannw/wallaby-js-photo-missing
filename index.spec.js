describe('wallaby', () => {
    it('should show the snapshot', async () => {
        const getPosts = async () => {
            throw new Error('12');
        };

        await expect(getPosts()).rejects.toThrowErrorMatchingSnapshot();
    });
});