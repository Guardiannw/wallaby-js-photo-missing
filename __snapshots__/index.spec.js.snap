// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wallaby should show the snapshot 1`] = `"1"`;
